charts/icm/README.asciidoc

```diff
@@ -1,99 +1,116 @@
 There is a naming convention for feature and bugfix branches like `feature/icm/<issue number>-<descriptive name>` or `bugfix/icm/<issue number>-<descriptive name>`.
 
-### Circular dependency
+=== Circular dependency
 
 There is a circular dependency to the underlying intershop project of this helm chart which manifests itself in having changes, which require also changes in the helm chart. To secure the version compatibility integration tests are done on pull requests before merging into the `main` branch.
 Some notes on this:
 * the `appVersion` in the `Chart.yaml` determines the compatible version to the current chart
-* the integration tests need published images on [intershophub](https://hub.docker.com/orgs/intershophub/repositories)
+* the integration tests need published images on https://hub.docker.com/orgs/intershophub/repositories[intershophub]
 * there might be changes in the underlying project which aren't published via image but chart changes are also required
-    * then either published dev-releases are needed or the feature branch couldn't be merged till an existing published release exists
-    * it's possible to do alpha/beta releases of helm charts, which could be used to test the base changes before publishing any image
+ * then either published dev-releases are needed or the feature branch couldn't be merged till an existing published release exists
+ * it's possible to do alpha/beta releases of helm charts, which could be used to test the base changes before publishing any image
 
-## Testing the Chart
+== Testing the Chart
 
-### helm-unit
+=== helm-unit
 
 There are helm-unit tests to support development and secure several functionality.
 
 Prerequisites are:
 
-* [helm-unittest](https://github.com/helm-unittest/helm-unittest)
-
-```bash
+* https://github.com/helm-unittest/helm-unittest[helm-unittest]
+[source,bash]
+----
 helm unittest --output-file unit.xml --output-type JUnit charts/icm
-```
+----
 
-### ct lint & install
+=== ct lint &amp; install
 
 Prerequisites are:
 
-* [kind cluster](https://github.com/kubernetes-sigs/kind)
+* https://github.com/kubernetes-sigs/kind[kind cluster]
 * Install cluster: `kind create cluster --config icm.yaml`
-
-```bash
+[source,bash]
+----
 docker run -it --network host --workdir=/data --volume <my kube config>:/root/.kube/config:ro --volume
 $(pwd):/data quay.io/helmpack/chart-testing:v3.8.0 ct lint --config ct_icm.yaml
-```
+----
 
-## Testing ICM
+== Testing ICM
 
 This helm chart also allows you to execute intershop htmlunit tests via a testrunner.
 
-### Execute locally
+=== Execute locally
+
 For the local test execution there are already preconfigured values-files orchestrated by a bash script.
 Follow these steps to execute a test:
 
-1. Be sure that each included chart is up-to-date:
-```bash
+.
```

Comment:
Is there something missing here? @skiesewetter-intershop

Reply:
Don't think so. I don't know where this dot comes from. I will remove it.